crates/ika-core/src/dwallet_mpc/dwallet_mpc_service.rs

```diff
@@ -296,6 +304,73 @@ impl DWalletMPCService {
         }
     }
 
+    /// Receive all completed MPC sessions from the MPC Output Verifier over the `consensus_round_completed_sessions` channel.
+    /// If the session exists, mark is as `MPCSessionStatus::Finished`.
+    /// Otherwise, create a new session with that status, in order to avoid re-running the computation for it.
```

Comment:
```suggestion
    /// Receive all completed MPC sessions from the MPC Output Verifier over the
    /// `consensus_round_completed_sessions` channel.
    /// If the session exists, mark is as [`MPCSessionStatus::Finished`].
    /// Otherwise, create a new session with that status, to avoid re-running the computation for it.
```